django_project/mezzanine_people/migrations/0001_initial.py

```diff
@@ -1,11 +1,9 @@
 # -*- coding: utf-8 -*-
-from south.utils import datetime_utils as datetime
-from south.db import db
-from south.v2 import SchemaMigration
-from django.db import models
+from django.db import models, migrations
```

Comment:
I could also not import south even after trying various ways to install it... replacing it with the modern django libraries worked a charm though!